src/from/dynamodb.js

```diff
@@ -4,7 +4,12 @@ import _ from 'highland';
 
 import { faulty } from '../utils';
 
-export const fromDynamodb = (event) => // eslint-disable-line import/prefer-default-export
+export const fromDynamodb = (event, opt = {
+  pkFn: 'pk',
+  skFn: 'sk',
+  discriminatorFn: 'discriminator',
+  eventTypePrefix: undefined,
+}) => // eslint-disable-line import/prefer-default-export
```

Comment:
Thanks, this solves my problem 👍 

There could be some edge cases defaulting to the object like that though

if I call 

```
fromDynamodb(event, { eventTypePrefix: '🐶'})
```
Then pkFn will be undefined causing an exception again accidentally. 

It could be safer to do 

```suggestion
export const fromDynamodb = (event, {
  pkFn = 'pk',
  skFn = 'sk',
  discriminatorFn = 'discriminator',
  eventTypePrefix = undefined,
} = {}) => // eslint-disable-line import/prefer-default-export
```

Reply:
i like the change. thx